canonical/library.canonical.com
# Implement caching and link debounce on navigation

## Done

- Implement TTL cache for the navigation
- Implements LRU cache of documents
- Throttles the link clicks in the navigation
- Changes navigation headings that don't have an associated page to italic
- Small refactor of js file structure and webpack

## QA

- Run the project locally (you will need to run a memcached server locally, on the default settings) and see that on the initial load of a page it is much slowly than a secondary load. You can add some logs into the `get_document_list` and `get_html` functions to ensure they are not being called a second time.

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3540
